src/seml/database.py

```diff
@@ -20,6 +22,19 @@ def get_collection(collection_name, mongodb_config=None, suffix=None):
 
 
 def get_mongo_client(db_name, host, port, username, password, **kwargs):
+    if 'ssh_config' in kwargs:
```

Comment:
can't we just add ssh_config as proper arg and set it to None by default?

Reply:
Hmm perhaps that is feasible